rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/60_empty.yml

```
@@ -0,0 +1,29 @@
+---
+"Empty Agg Body":
```

Super nit, we return empty `aggs`:

```suggestion
"Empty aggs Body":
```